public/manifest.json

```diff
@@ -2,21 +2,6 @@
   "short_name": "React App",
   "name": "Create React App Sample",
   "icons": [
-    {
```

Comment:
`

Reply:
Na przyszłość takie rzeczy warto robić w osobnym branchu żeby się nie zgubiło gdzie to było zrobione